### ISSUE TITLE
XIgnite quote service has expired

### ISSUE THREAD

**Issue by dpinto-pivotal** [dpinto-pivotal/cf-SpringBootTrader#40]
account and token for use of XIgnote service has expired.

We shouldn't really be using services that we need to register for, or that there are expiration dates or low threshold limits of use.

Need to add to the docs on how to create account and change token for now, but look at using different service - specially now that we have yahoo quotes as well.

**Comment by dpinto-pivotal** [comment]
Is anyone coming up against this issue?

**Comment by svrc-pivotal** [comment]
yup.   I've tried signing up for a trial but it's not letting me even get quotes for the trial.

**Comment by kelapure** [comment]
Instead of XIgnite can we use something like this `http://www.google.com/finance/info?q=NASDAQ%3aGOOG`

**Comment by dpinto-pivotal** [comment]
yes we do need to get rid of xignite service. we have marketitondemand and yahoo. is there documentation on the google one?
